controllers/getCommentsById.controller.js

```diff
@@ -0,0 +1,20 @@
+const fetchCommentsById = require('../models/fetchCommentsById.model');
+const fetchReviewById = require('../models/fetchReviewById.models');
+
+function getCommentsById(req, resp, next) {
+    const { review_id } = req.params;
+
+    // check the review_id is associated with a review
+    fetchReviewById(review_id).then((review) => {
+        // if there is a review, then fetch the comments
+        fetchCommentsById(review_id).then((comments) => {
```

Comment:
Remember to return this promise and avoid a nested .then
This will also allow you to have a single .catch - multiple is a bit tricky to keep track of

Reply:
Managed this one now! :)